docs/articles/documentation/guides/basic-guides/interact-with-the-page.md

```diff
@@ -333,3 +333,41 @@ Mouse event | Touch event
 `mousemove` (when hovering or dragging) | `touchmove` (dragging only)
 `mousedown` | `touchstart`
 `mouseup`   | `touchend`
+
+## Interaction Requirements
+
+To be properly targeted by TestCafe, elements have to satisfy several conditions:
```

Comment:
We should be very careful when we use adjectives in technical texts.
When writing docs, it's appropriate to ask this boring nerdy question: "Can they be improperly targeted?"
So, we'd better say 

*TestCafe actions can interact with elements if they satisfy the following conditions:*